adafruit_platformdetect/board.py

```diff
@@ -25,8 +25,6 @@
 GENERIC_LINUX_PC            = "GENERIC_LINUX_PC"
 PYBOARD                     = "PYBOARD"
 NODEMCU                     = "NODEMCU"
-ORANGE_PI_PC                = "ORANGE_PI_PC"
```

Comment:
You'll want to leave both of these IDs here too. This is where they are defined.